contrib/gitian-descriptors/gitian-linux.yml

```
@@ -122,7 +122,7 @@ script: |
   done
 
   cd bitcoin
-  BASEPREFIX=`pwd`/depends
+  BASEPREFIX=$(pwd)/depends
```

Let's be consistent and just use `$PWD` everywhere. 
```suggestion
  BASEPREFIX="${PWD}/depends"
```

---

> Let's be consistent and just use `$PWD` everywhere.

Agree.

Ref:
```bash
$ echo $0
bash
$ type pwd
pwd is a shell builtin
$ pwd --help
pwd: pwd [-LP]
    Print the name of the current working directory.
    
    Options:
      -L	print the value of $PWD if it names the current working
    		directory
      -P	print the physical directory, without any symbolic links
    
    By default, `pwd' behaves as if `-L' were specified.
    
    Exit Status:
    Returns 0 unless an invalid option is given or the current directory
    cannot be read.
```

Going to fix tonight.

---

Done.